src/main.rs

```diff
@@ -3,50 +3,51 @@
 #![feature(abi_avr_interrupt)]
 
 use arduino_hal::default_serial;
-use mesh_lib::{init_node, AddressType, LifeTimeType, NodeConfig, NodeString};
+use mesh_lib::{AddressType, LifeTimeType, Node, NodeString};
 use panic_halt as _;
 
 mod mesh_lib;
 
 use ufmt::uwrite;
 
 use platform_millis_arduino_nano::{init_timer, ms, Atmega328pTime, PlatformTime};
+use platform_serial_arduino_nano::{init_serial, ArduinoNanoSerial};
 
 #[arduino_hal::entry]
 fn main() -> ! {
     let dp = arduino_hal::Peripherals::take().unwrap();
     let pins = arduino_hal::pins!(dp);
 
     init_timer(dp.TC0);
+    init_serial(default_serial!(dp, pins, 9600));
 
-    let mut mesh_node = init_node(NodeConfig {
-        device_identifier: 1 as AddressType,
-        listen_period: 150 as ms,
-        usart: default_serial!(dp, pins, 9600),
-    });
+    let mut mesh_node = Node::new(1 as AddressType, 150 as ms);
```

Comment:
this now looks less transparent - what is 1, what is 150 ms?

Reply:
@berkus Got it. LL turn it back